app/Utils/CheckoutHandler.php

```diff
@@ -122,6 +122,7 @@ public function userTransactionsNotInCheckout(array $payment_types)
         $payment_type_ids = $this->paymenyTypeIDs($payment_types);
 
         $user_transactions_not_in_checkout = Transaction::where('receiver_id', Auth::user()->id)
+            ->with(['type', 'payer'])
```

Comment:
I think we only use the payer's name. Can we say `payer:id,name` if it's possible that the payer is null sometimes?

Reply:
Absolutely! It just adds the columns to the select statement:
`select id, name from users where users.id in (...) and verified = 1`
The difference here is probably negligible but you should definitely use this if you have a model with lots of attributes or attributes with huge amount of data (text, json).

Reply:
Ah, I see, thanks!
BTW, are you using some tools to check how many queries are run for each Laravel page? Or do you have any suggestions on how to check performance?